jetcd-core/src/test/java/io/etcd/jetcd/WatchUnitTest.java

```diff
@@ -98,12 +100,14 @@ public void testCreateWatcherAfterClientClosed() {
     watchClient.close();
 
     assertThatExceptionOfType(ClosedClientException.class)
-        .isThrownBy(() -> watchClient.watch(KEY, Watch.listener(r -> {})));
+        .isThrownBy(() -> watchClient.watch(KEY, Watch.listener(r -> {
```

Comment:
I am unsure if this formatting is necessary, I general like to have a minimum changes in a PR.

Reply:
Restored newlines, but would leave spaces formatting, it looks like it formatted it correctly.